### PR TITLE
dev-php/xdebug: Bump 2.6.0-alpha1

### PR DESCRIPTION
**Pull request by CharlesNerot**
Bug: https://bugs.gentoo.org/640256
Package-Manager: Portage-2.3.19, Repoman-2.3.6

### CODE REVIEW

**Comment by CharlesNerot** [comment]
ping @gentoo/php

File: dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+EAPI=6
+
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~x86"
+
+MY_PV="${PV/_/}"
+MY_PV="${MY_PV/rc/RC}"
+
+DESCRIPTION="Xdebug client for the Common Debugger Protocol (DBGP)"
+HOMEPAGE="http://www.xdebug.org/"
+SRC_URI="http://pecl.php.net/get/xdebug-${MY_PV}.tgz"
```
Comment (SoapGentoo) [inline dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild]: free line after `SRC_URI`

File: dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+LICENSE="Xdebug"
+SLOT="0"
+IUSE="libedit"
```
Comment (SoapGentoo) [inline dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild]: `KEYWORDS` goes between `SLOT` and `IUSE`

File: dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+S="${WORKDIR}/xdebug-${MY_PV}/debugclient"
+
+DEPEND="libedit? ( dev-libs/libedit ) net-libs/libnsl:0="
```
Comment (SoapGentoo) [inline dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild]: one dependeny per line please

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```
Comment (SoapGentoo) [inline dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: free line after `EAPI=6` (remove quotes too)

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+PHP_EXT_NAME="xdebug"
+PHP_EXT_INI="yes"
+PHP_EXT_ZENDEXT="yes"
+PHP_EXT_INIFILE="2.4.1-xdebug.ini"
+
+USE_PHP="php7-2 php7-1 php7-0"
+
+MY_PV="${PV/_/}"
+MY_PV="${MY_PV/rc/RC}"
```
Comment (SoapGentoo) [inline dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: put these after the `inherit` line

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+S="${WORKDIR}/${PN}-${MY_PV}"
```
Comment (SoapGentoo) [inline dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: after `RDEPEND`

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+inherit php-ext-source-r3
+
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~x86"
```
Comment (SoapGentoo) [inline dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: same here, between `SLOT` and `IUSE`

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+DESCRIPTION="A PHP debugging and profiling extension"
+HOMEPAGE="http://www.xdebug.org/"
+SRC_URI="http://pecl.php.net/get/${PN}-${MY_PV}.tgz"
```
Comment (SoapGentoo) [inline dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: empty line after `SRC_URI`

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+LICENSE="Xdebug"
+SLOT="0"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}
+		~dev-php/xdebug-client-${PV}"
```
Comment (SoapGentoo) [inline dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: too much indentation (one tab only)

File: dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild (hunk 1 above)
Comment (CharlesNerot) [reply dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild (hunk 2 above)
Comment (CharlesNerot) [reply dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild (hunk 3 above)
Comment (CharlesNerot) [reply dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild (hunk 4 above)
Comment (CharlesNerot) [reply dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild (hunk 5 above)
Comment (CharlesNerot) [reply dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild (hunk 6 above)
Comment (CharlesNerot) [reply dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild (hunk 7 above)
Comment (CharlesNerot) [reply dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild (hunk 8 above)
Comment (CharlesNerot) [reply dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild (hunk 9 above)
Comment (CharlesNerot) [reply dev-php/xdebug/xdebug-2.6.0_alpha1.ebuild]: done

File: dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+DESCRIPTION="Xdebug client for the Common Debugger Protocol (DBGP)"
+HOMEPAGE="http://www.xdebug.org/"
+SRC_URI="http://pecl.php.net/get/xdebug-${MY_PV}.tgz"
+
+LICENSE="Xdebug"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~x86"
+IUSE="libedit"
+
+S="${WORKDIR}/xdebug-${MY_PV}/debugclient"
```
Comment (SoapGentoo) [inline dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild]: in the other ebuild you did it correctly and put it after `RDEPEND`, please do it here too. Beyond that, looks good

Comment (CharlesNerot) [reply dev-php/xdebug-client/xdebug-client-2.6.0_alpha1.ebuild]: done

**Pull request closed without merging**